lupa/__init__.py

```diff
@@ -68,7 +67,8 @@ def __getattr__(name):
             # "from lupa import lua54" etc.
             assert name not in globals()
             try:
-                module = __import__(name, globals=globals(), locals=locals(), level=1)
+                with eager_global_linking():
+                    module = __import__(name, globals=globals(), locals=locals(), level=1)
```

Comment:
I actually wonder now why I'm using two different sets of arguments to call `__import__()`. :-/ I'll fix that up once this is merged. The original one was probably fine.